libs/dh/shared/feature-authorization/src/lib/scope.service.ts

```diff
@@ -16,18 +16,105 @@
  */
 
 import { Inject, Injectable } from '@angular/core';
+import { MsalBroadcastService, MsalService } from '@azure/msal-angular';
+import { EventMessage, EventType } from '@azure/msal-browser';
 import {
   DhB2CEnvironment,
   dhB2CEnvironmentToken,
 } from '@energinet-datahub/dh/shared/environments';
+import { DhFeatureFlagsService } from '@energinet-datahub/dh/shared/feature-flags';
+import { filter } from 'rxjs';
+import { ScopeStorage, scopeStorageToken } from './scope-storage.service';
+
+export const scopesKey = 'actor-scopes';
+export const scopeKey = 'active-actor-scope';
```

Comment:
suggestion: let's make this variable names a bit more explicit. Consider `actorScopesKey` and `activeActorScopeKey`.

Reply:
fixed